src/components/App.jsx

```diff
@@ -119,9 +120,13 @@ class App extends Component {
   }
 
   render() {
-    // Send performance event on initial render
+    // Send event on initial render
     if (!this.state.hasPreviouslyRendered) {
       utils.markPerformanceIfAble('Payment app began painting');
+      sendTrackEvent(
```

Comment:
A concern I have here is that the `identify` call might not have been made yet, in which case we can't have any segment events here